lib/Image/Adapter/Imagick.php

```diff
@@ -143,21 +143,25 @@ public function load($imagePath, $options = [])
                         // - it seems that -clip doesnt work with the ImageMagick version
                         //   ImageMagick 6.9.7-4 Q16 x86_64 20170114 (which is used in Debian 9)
                         // - Imagick 3.4.4 with ImageMagick 7 on OSX has horrible broken clipping support
-                        $i->setImageAlphaChannel(\Imagick::ALPHACHANNEL_TRANSPARENT);
-                        $i->clipImage();
+
+                        if(in_arrayi($this->sourceImageFormat, ['jpeg', 'pjpeg', 'jpg'])) {
```

Comment:
```suggestion
                        if(in_arrayi($this->sourceImageFormat, ['jpeg', 'pjpeg', 'jpg', 'pjpeg', 'jpe'])) {
```

Reply:
But this does not solve the wrong display on your system, does it?

Reply:
@BlackbitNeueMedien exactly, that was just a missing format I noticed before I did the test 😊
Must be related to the underlying ImageMagick version, because our Imagick (PHP extension) version is actually the same (`3.4.4`). 
So maybe we need to somehow utilize `Imagick::getVersion()` to fix that?

Reply:
I already tried. The problem is that I have 6.9.7-4, you have 6.9.10-23 and I have also tried it with 7.0.10-25. The problem now is that it works with this PR in 6.9.7-4 and 7.0.10-25 but not in your 6.9.10-23. So it will be difficult to check with `version_compare()`.
So the problem seems to be what @ssiergl said in https://github.com/pimcore/pimcore/issues/5505#issuecomment-673386221 
> Imagemagick quite often breaks clipping. Aside that there are some broken imagemagick 6.9 Versions (like used in debian 9) it is pretty often broken in imagemagick 7 as well (E.g. right now the last eight stable releases are broken again)

Reply:
@BlackbitNeueMedien ok, agree. 
I'd then like to propose the following solution: 
For 6.9 we leave everything as it is, so we don't change any behavior. 

For X: 
We implement it as the expected behavior of those IM clipping functions is and we just ignore the bugs of ImageMagick. 
Additionally we refactor `\Pimcore\Image` to return non shared services for either `Imagick` or `GD`. 
Then it's easy to override `Imagick` including some fixes for the individual broken ImageMagick version. 
Wdyt?